app/mailers/signup_mailer.rb

```diff
@@ -0,0 +1,16 @@
+class SignupMailer < ApplicationMailer
+  default from: 'spacex@rails.com'
+
+  def welcome_email(user)
+    @name = user.first_name
+    mail to: user.email, subject: 'Welcome to My Awesome Site'
+  end
+
+  def new_organization_created_email(user)
+    @super_admin = User.find_by(role: "super_admin")
+    @organization = user.own_organization
+    @organization_name = @organization.name
+    @name = user.first_name
```

Comment:
try to avoid such  specific instance variables.. just set `@user = user` and get data thru it